### DISCUSSION TITLE
Click the adjustment window in the full-screen state of the agent terminal, and the picture will be clearer

### DISCUSSION THREAD

**Discussion by youhers** [rustdesk/rustdesk#5717] [Feature Request]
Click the adjustment window in the full-screen state of the agent terminal, the picture will be clearer, but after double-clicking the dialog box border, the picture will become very unclear

![微信截图_20230917103641](https://github.com/rustdesk/rustdesk/assets/106003909/63338357-588f-47a1-93eb-f7e12b4ba407)
![微信图片_20230917103716](https://github.com/rustdesk/rustdesk/assets/106003909/86933bad-16cd-4776-b8f3-66acda4588c2)

**Comment by rustdesk** [comment]
recording video please. Frankly, no idea what you are saying per pictures and text. Only seeing the first picture is clearer.

**Comment by youhers** [comment]
https://github.com/rustdesk/rustdesk/assets/71636191/e3820155-2029-4f27-b9bf-06bf18a7c3c3


![image](https://github.com/rustdesk/rustdesk/assets/71636191/49ccee1d-4758-48dd-871c-f540d1444908)


https://github.com/rustdesk/rustdesk/assets/71636191/eb252ea3-9bbc-4df1-8d49-bdd2a6c9a230

**Reply by rustdesk** [reply]
upload failed

**Reply by youhers** [reply]
Can I email you, the proxy network quality is not good

**Reply by youhers** [reply]
This problem occurs when the win screen is zoomed in, and when the picture is zoomed out, there is no problem, but there is still a problem, and I will email you

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/pull/5721

**Reply by dignow** [reply]
The blurry image issue is still not fixed.

**Reply by rustdesk** [reply]
Ok, continue.

**Reply by dignow** [reply]
> The blurry image issue is still not fixed.

https://github.com/rustdesk/rustdesk/pull/5749
